dspace/config/dspace.cfg

```diff
@@ -1429,6 +1429,9 @@ webui.content_disposition_threshold = 8388608
 # the directory where the generated sitemaps are stored
 sitemap.dir = ${dspace.dir}/sitemaps
 
+# sitemaps endpoint path used in SitemapRestController and in the url paths when sitemaps are generated
+sitemap.path = sitemaps
```

Comment:
Minor thing here. I'd recommend we comment this out by default and change comment slightly. So, update to something like:

```
# Customize the path of sitemaps in the server webapp
# Defaults to "sitemaps", which means the sitemaps are available at ${dspace.server.url}/sitemaps
# sitemap.path = sitemaps
```